linkis-engineconn-plugins/spark/src/main/scala/org/apache/linkis/engineplugin/spark/config/SparkConfiguration.scala

```diff
@@ -50,6 +50,18 @@ object SparkConfiguration extends Logging {
   val SPARK_APP_RESOURCE = CommonVars[String]("spark.app.resource", "")
   val SPARK_APP_CONF = CommonVars[String]("spark.extconf", "")
 
+  val SPARK_K8S_CONFIG_FILE = CommonVars[String]("spark.k8s.config.file", "")
```

Comment:
If it is not a native parameter of spark or k8s, it need to add links prefix

Reply:
Ok, thanks for reviewing the code